scripts/mixins/oldjavawarning.zs

```diff
@@ -0,0 +1,25 @@
+#loader mixin
+
+#mixin Mixin
+#{targets: net.darkhax.oldjava.OldJavaWarning}
+zenClass MixinOldJavaWarning {
+    #mixin Static
+    #mixin Redirect
+    #{
+    #    method: "checkJavaVersion",
+    #    at: {
+    #        value: "INVOKE",
+    #        target: "Ljava/lang/String;compareTo(Ljava/lang/String;)I"
+    #    }
+    #}
+    function fixJavaVersionCheck(min as string, current as string) as int {
+        val currentSplit as string[] = current.split("_");
+        val minSplit as string[] = min.split("_");
+        if (currentSplit.length > 1 && minSplit.length > 1 && currentSplit[1].matches("[0-9]+") && minSplit[1].matches("[0-9]+")) {
+            val currentVersion as int = currentSplit[1] as int;
+            val minVersion as int = minSplit[1] as int;
+            return minVersion > currentVersion;
```

Comment:
Its not an issue, im just curious.
Isnt this return should cause return type exception? Function is `int` but there we returning `bool` value. Or functions in mixins can cast return values automatically?

Reply:
Oh, that was my mistake. But it works? Maybe it is because actually bool value doesn't exist in java virtual machine, in fact, they are 0 or 1 int value.